README.md

```diff
@@ -24,6 +24,10 @@ Truffle: v5.1.31
 
 Solc: 0.6.4+commit.1dca32f3
 
+Tips: You can manage multi version of 
+- solc by [solc-select](https://github.com/crytic/solc-select).
```

Comment:
Could we add an example command e.g. `solc-select install 0.6.4 && solc-select use 0.6.4` ? 
Similar for nvm.

Reply:
sure, updated.